Mathlib/Analysis/Complex/Circle.lean

```diff
@@ -182,6 +182,17 @@ protected lemma norm_smul {E : Type*} [SeminormedAddCommGroup E] [NormedSpace 
     ‖u • v‖ = ‖v‖ := by
   rw [Submonoid.smul_def, norm_smul, norm_eq_of_mem_sphere, one_mul]
 
+lemma exp_nat_mul (n : ℕ) (t : ℝ) : exp (n * t) = exp t ^ n := by
+  simp only [exp, ContinuousMap.coe_mk, ofReal_mul, ofReal_natCast, mul_assoc, Complex.exp_nat_mul]
```

Comment:
If you move this to after `probChar` is defined, then there's a quicker proof `simpa using Real.probChar.map_nsmul_eq_pow n t`.